### DISCUSSION TITLE
【求助】 这类有加密水印的视频无法抓取

### DISCUSSION THREAD

**Discussion by Jeffreywei99** [xifangczy/cat-catch#308] [Q&A]
求助，有一些视频，有这种打了加密水印的无法抓取。 不能嗅探到，也不能用缓存捕捉。 这是什么原因？ 可以改进么
![1111111111](https://github.com/xifangczy/cat-catch/assets/161019818/41054ae1-a63b-42e2-a049-fb320532c472)

**Comment by xifangczy** [comment]
需要提供播放网址以便分析

**Reply by Jeffreywei99** [reply]
是需要登录的网站，我有密码。这类网址是否方便分析呢，我私信发给你。

**Reply by Jeffreywei99** [reply]
已发送您gmail邮箱
